features/fixtures/sidekiq/Dockerfile

```diff
@@ -1,6 +1,9 @@
 ARG RUBY_VERSION
 FROM ruby:$RUBY_VERSION
 
+WORKDIR /bugsnag
```

Comment:
This change should really be applied to all features in a separate PR

Reply:
This is currently in progress

Reply:
Where? When will it be done?

Reply:
I'm just running the maze tests, and will PR them once I'm sure they're running correctly